airbyte-integrations/connectors/destination-iceberg/src/test-integration/java/io/airbyte/integrations/destination/iceberg/nessie/BaseIcebergNessieCatalogS3IntegrationTest.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.integrations.destination.iceberg.nessie;
+
+import static io.airbyte.integrations.destination.iceberg.IcebergIntegrationTestUtil.ICEBERG_IMAGE_NAME;
+
+import java.util.HashSet;
+import java.util.List;
+import io.airbyte.cdk.integrations.standardtest.destination.DestinationAcceptanceTest;
+import io.airbyte.integrations.destination.iceberg.IcebergIntegrationTestUtil;
+import io.airbyte.integrations.destination.iceberg.config.format.DataFileFormat;
+import io.airbyte.integrations.destination.iceberg.container.RESTServerWithMinioCompose;
+
+import org.junit.jupiter.api.AfterAll;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.fasterxml.jackson.databind.JsonNode;
+
+public abstract class BaseIcebergNessieCatalogS3IntegrationTest extends DestinationAcceptanceTest {
+
+  private static final Logger LOGGER = LoggerFactory.getLogger(BaseIcebergNessieCatalogS3IntegrationTest.class);
+
+  private static RESTServerWithMinioCompose composeContainer;
+  private static JsonNode config;
+
+  static void startCompose(final DataFileFormat fileFormat) {
+    composeContainer = new RESTServerWithMinioCompose();
+    composeContainer.start();
+    config = composeContainer.getComposeConfig(fileFormat);
+    IcebergIntegrationTestUtil.createS3WarehouseBucket(config);
+    LOGGER.info("==> Started REST Server with Minio - Docker Compose...");
+  }
+
+  @AfterAll
+  public static void stopCompose() {
+    IcebergIntegrationTestUtil.stopAndCloseContainer(composeContainer, "REST Server with Minio - Docker Compose");
```

Comment:
Please check this class, I think Integration test is running with Rest server. Not the Nessie.